tests/Aql/EngineInfoContainerCoordinatorTest.cpp

```diff
@@ -146,7 +148,21 @@ TEST(EngineInfoContainerTest,
       scopeGuard([=]() noexcept { ServerState::instance()->setRole(oldRole); });
 
   // simon: we only use this query for the API
-  auto q = server.createFakeQuery("RETURN 1");
+  VPackBuilder queryOptions;
+  queryOptions.openObject();
+  queryOptions.close();
+
+  auto& database = server.getFeature<DatabaseFeature>();
+  auto system = database.lookupDatabase(StaticStrings::SystemDatabase);
+  TRI_ASSERT(system != nullptr);
+  auto ctx = transaction::StandaloneContext::Create(*system);
+  transaction::Methods trx(ctx);
+  trx.addHint(transaction::Hints::Hint::GLOBAL_MANAGED);
+  trx.begin();
+  auto q = aql::Query::create(ctx, aql::QueryString(std::string("RETURN 1")),
+                              nullptr, aql::QueryOptions(queryOptions.slice()));
```

Comment:
Instead of creating a VPackBuilder with an empty object in it, this would do it:
```suggestion
                              nullptr, aql::QueryOptions(VPackSlice::emptyObjectSlice()));
```